samples/core/kfp_env_validation/kfp_env_validation.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>In the doc string can you clarify the meaning of the return value. For example, return True only when configured properly.</p><p> </p><p>Also, I cannot find the API enable link using <code>project_id</code>. Did I miss it?</p><p> </p><p>P.S.: Reading from below it seems to me that we're not really using the return value.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
good point, I was originally planning to use the return value for pipeline execution logic, though I don't think it is needed as if the step fails it stops the execution. Removing the return values accordingly.

Reply:
for API enabled linked using project_id it is under step 3             

`error_list.append('API \"%s\" is not enabled. To enable this api go to https://pantheon.corp.google.com/apis/library/%s?project=%s' %(api,api,project_id))`

Reply:
just realized I only need it under step 3... removing it from other steps.

Reply:
also good point , updating comments on other steps ... project id is used for logging / error message there.